### DISCUSSION TITLE
Lost email verification and password

### DISCUSSION THREAD

**Discussion by prashanthaakula** [community/community#78671] [New to GitHub]
One of my team member lost their password and when they try to reset, getting an error to verify the email. However, the email verification link was expired as person look at that after 2 months. Howe can we get email verified and reset the password ?

**Comment by erickdc7** [comment]
Here's what you can do to resolve this:

1. **Contact Support**:
   Reach out to the support team of the service or platform where the account is registered. They can assist with re-sending a new verification link or guiding you through the process of verifying the email and resetting the password.

2. **Check Spam/Junk Folder**:
   Sometimes, verification emails may be mistakenly filtered into the spam or junk folder. Ask your team member to check these folders for any missed emails from the service.

3. **Request a New Verification Link**:
   If the platform allows, your team member can try requesting a new verification link through the platform's user interface. There's usually an option like "Resend verification email" or "Didn't receive the email?".

4. **Update Email Address**:
   If the email address is no longer accessible, your team member may need to update their email address on the account, if possible. This might also require contacting support for assistance.

5. **Ensure Accurate Email Entry**:
   When requesting a new verification link or password reset, ensure that the correct email address is entered without any typos.

6. **Wait for the Link**:
   After requesting a new link, it may take a few minutes to arrive. Patience is key, and it's important not to request multiple links in quick succession as this may cause delays.

Remember, for security reasons, the support team may ask for additional information to verify the identity of the account holder before assisting with the email verification and password reset. It's important to provide accurate information to facilitate the process.
